workspaces/noop/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@internal/noop",
+  "private": true,
+  "engines": {
+    "node": "18 || 20"
```

Comment:
Can we get away with not having this? Just means something we have to update

Reply:
I think it's fine as it is consistent with other workspaces. The day we will update, we'll touch all of them anyway